File: app/src/main/java/com/worldventures/dreamtrips/social/ui/profile/service/analytics/ProfileUploadingAnalyticAction.java
package com.worldventures.dreamtrips.social.ui.profile.service.analytics;

import com.worldventures.janet.analytics.AnalyticsEvent;
import com.worldventures.core.service.analytics.ApptentiveTracker;
import com.worldventures.core.service.analytics.BaseAnalyticsAction;

@AnalyticsEvent(action = "profile", category = "nav_menu", trackers = ApptentiveTracker.TRACKER_KEY)
public class ProfileUploadingAnalyticAction extends BaseAnalyticsAction {

}

File: app/src/main/java/com/worldventures/dreamtrips/social/service/profile/analytics/ProfileUploadingAnalyticAction.kt
package com.worldventures.dreamtrips.social.service.profile.analytics

import com.worldventures.core.service.analytics.ApptentiveTracker
import com.worldventures.core.service.analytics.BaseAnalyticsAction
import com.worldventures.janet.analytics.AnalyticsEvent

@AnalyticsEvent(action = "profile", category = "nav_menu", trackers = [(ApptentiveTracker.TRACKER_KEY)])
class ProfileUploadingAnalyticAction : BaseAnalyticsAction()
